### PR TITLE
Apply

### PR DESCRIPTION
**Pull request by daisycrego**

### CODE REVIEW

File: apply/views.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from django.shortcuts import render
+from django.urls import reverse
+from django.contrib import messages
+from jobs.models import Job
+from .forms import JobApplicationForm
+from .models import Application
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+
```
Comment (surajgovardhangaikwad) [inline apply/views.py]: We should add @login_required here

File: teamone/urls.py.orig [hunk 2]
```diff
@@ -0,0 +1,62 @@
+"""teamone URL Configuration
```
Comment (surajgovardhangaikwad) [inline teamone/urls.py.orig]: I don't think this file should even be in git? 
I may be wrong. 
Either way, there are some unresolved merge conflicts here

File: apply/tests/tests.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+from apply.forms import JobApplicationForm
+from apply.models import Application
+from jobs.models import Job
+from tests.tests import valid_data, invalid_data
+from uplyft.models import CustomUser
+from django.contrib.auth import get_user_model
+from django.contrib.auth import authenticate, login
+from selenium import webdriver
+
+class ApplyViewTests(TestCase):
```
[the remark this answers is not in the document]
Comment (daisycrego) [reply apply/tests/tests.py]: Done

File: apply/admin.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
[the remark this answers is not in the document]
Comment (daisycrego) [reply apply/admin.py]: done

File: apply/views.py (hunk 1 above)
Comment (daisycrego) [reply apply/views.py]: Done

File: teamone/urls.py.orig (hunk 2 above)
Comment (daisycrego) [reply teamone/urls.py.orig]: done

**Pull request merged**